src/main/java/seedu/address/ui/ProjectCard.java

```diff
@@ -89,5 +94,11 @@ public ProjectCard(Project project, int displayedIndex) {
                 deadline.setStyle("-fx-text-fill: green;");
             }
         }
+
+        // Set the completion status
+        completionStatusText.setText("Completed?:");
+        completionStatus.setSelected(project.getCompletionStatus().isCompleted);
+        // Prevent user from interacting with the checkbox through GUI
+        completionStatus.setDisable(true);
```

Comment:
You could have these fields as attributes in the FXML files so that they are even more explicit. This is how I implemented the checkbox for tasks which I will PR in soon.


                <CheckBox fx:id="isDone" disable="true" styleClass="task_label"/>

Reply:
Noted, I could change it in the future.